src/services/service.ts

```diff
@@ -292,12 +293,13 @@ export function getWorkflowList(appId, filteredEnvIds?: string) {
     return get(URL)
 }
 
-export function getWorkflowViewList(appId, filteredEnvIds?: string) {
-    let filteredEnvParams = ''
-    if (filteredEnvIds) {
-        filteredEnvParams = `?envIds=${filteredEnvIds}`
+export function getWorkflowViewList(appId, filteredEnvIds?: string, offending: `policy/${PolicyKindType}` = null) {
```

Comment:
Since we are sending policy can we rename it to policyKind or something generic can sync with BE as well